chore: Migration from the Play Core Java

ryaa opened the pull request:
Changes:
- migrate from the google play SDK to avoid warning from google play when the app is targeting SDK 34+ (targetSdkVersion). For more details please see https://github.com/pushandplay/cordova-plugin-apprate/issues/332

ryaa committed:
chore: Migration from the Play Core Java

Changes:
- migrate from the google play SDK to avoid warning from google play when the app is targeting SDK 34+ (targetSdkVersion). For more details please see https://github.com/pushandplay/cordova-plugin-apprate/issues/332

westonganger commented on package.json:
```diff
@@ -1,6 +1,6 @@
 {
   "name": "cordova-plugin-apprate",
-  "version": "1.7.3",
```
Can you please remove the version bump from this PR. Other than that this PR looks good.

westonganger replied:
Also can you please add a changelog entry

ryaa replied:
@westonganger I have done the required changes, plus updated README to reference Play In-App Reviews Library instead of Google Play Gore

ryaa committed:
chore: Migration from the Play Core Java

Changes:
- reverted the version bump
- added a changelog entry
- updated README to reference Play In-App Reviews Library instead of Google Play Gore

westonganger commented on CHANGELOG.md:
```diff
@@ -3,6 +3,9 @@
   - Nothing yet
 
+- v1.7.4
+  - [PR #334](https://github.com/pushandplay/cordova-plugin-apprate/pull/334) - Migration from the Play Core Java
```
Move this under unreleased replacing nothing yet. Also remove the reference to 1.7.4

ryaa replied:
done

westonganger commented on plugin.xml:
```diff
@@ -19,7 +19,7 @@ specific language governing permissions and limitations
 -->
 
-<plugin xmlns="http://cordova.apache.org/ns/plugins/1.0" id="cordova-plugin-apprate" version="1.7.3">
+<plugin xmlns="http://cordova.apache.org/ns/plugins/1.0" id="cordova-plugin-apprate" version="1.7.4">
```
Same here don't bump the version

ryaa replied:
done

ryaa committed:
chore: Migration from the Play Core Java

Changes:
- updated changelog
- reverted the version bump in plugin.xml file

westonganger commented:
This change is now released in v1.8.0

ryaa commented:
> This change is now released in v1.8.0

It seems that the release notes has not been added https://github.com/pushandplay/cordova-plugin-apprate/releases

westonganger commented:
@ryaa thanks for pointing that out. The release notes are in the CHANGELOG.md file. But I do like to create git tags and Github releases too, so thats been added now.
